src/test/java/microsoft/exchange/webservices/data/core/EwsUtilitiesTest.java

```diff
@@ -35,14 +34,4 @@
   public void testGetBuildVersion() {
     Assert.assertEquals("Build version must be 0s", "0.0.0.0", EwsUtilities.getBuildVersion());
```

Comment:
Retuning `0.0.0.0` as the current build-version might be wrong. Maybe we should read a property here which can be filled by travis with every build for instance. Or we should remove this entirely.